roles/node/templates/testnet/validator/config.yml.j2

```diff
@@ -0,0 +1,49 @@
+---
+authority-key-pair:
+  path: {{ node_authority_key_path }}
+protocol-key-pair:
+  path: {{ node_protocol_key_path }}
+network-key-pair:
+  path: {{ node_network_key_path }}
+account-key-pair:
+  path: {{ node_account_key_path }}
+db-path: {{ node_authorities_data_dir }}
+network-address: /ip4/0.0.0.0/tcp/{{ iota.validator.ports.network_address_port }}/http
+metrics-address: "0.0.0.0:{{ iota.validator.ports.metrics_port }}"
+admin-interface-address: "127.0.0.1:{{ iota.validator.ports.admin_interface_port }}"
+consensus-config:
+  db-path: {{ node_consensus_data_dir }}
+  internal-worker-address: null
+enable-event-processing: false
+p2p-config:
```

Comment:
include this https://github.com/encapsulate-xyz/iota-ansible/pull/1/commits/48d26549c5e52829958f944e092224895cd2d2ae#diff-ac0c5e498fb3b16dce6ef4a2c3d97480e3df082ce34370616c77b87160771e6dR24 here as well